### ISSUE TITLE
no-member error raised when access ForeignKey's second field

### ISSUE THREAD

**Issue by schemacs** [PyCQA/pylint-django#110]
Accessing the first name field of Author in Book is ok, but when accessing the second name field, it reports no-member error.

```python
"""sample"""
from django.db import models


class Author(models.Model):
    """Author model"""
    name = models.CharField(max_length=200)
    name2 = models.CharField(max_length=200)


class Book(models.Model):
    """Book model"""
    author = models.ForeignKey('Author', on_delete=models.PROTECT)

    def author_name(self):
        """author name"""
        return self.author.name

    def author_name2(self):
        """author name2"""
        return self.author.name2
```

```bash
$ pylint --load-plugins=pylint_django --reports=n models.py
E: 21,15: Instance of 'ForeignKey' has no 'name2' member; maybe 'name'? (no-member)

------------------------------------------------------------------
Your code has been rated at 5.00/10 (previous run: 5.00/10, +0.00)
$ pylint --version
pylint 1.7.5,
astroid 1.5.3
Python 3.6.4 (default, Dec 22 2017, 09:14:27)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
$ pip3 freeze | grep pylint-django
pylint-django==0.7.2
```

**Comment by atodorov** [comment]
The error is raised when the FK model is specified as a string, rather than class name identifier.

**Comment by hemanthsp** [comment]
so how to fix this issues, and what to change in the code
